project/ValidatePullRequest.scala

```diff
@@ -143,7 +143,7 @@ object ValidatePullRequest extends AutoPlugin {
 
       // TODO could use jgit
       log.info(s"Comparing [$targetId] with [$prId] to determine changed modules in PR...")
-      val gitOutput = "git diff %s..%s --name-only".format(targetId, prId).!!.split("\n")
+      val gitOutput = "git diff HEAD^ --name-only".!!.split("\n")
```

Comment:
This is great! `targetId` is not used anymore and can be removed with corresponding settings.